client/src/components/Home.vue

```diff
@@ -3,9 +3,14 @@
     h1 Home
     ul(v-if="results")
       li(v-for="item in results.results", v-on:click="goToPlaylist(item)")
-        img(v-if="item.images[0]", :src="item.images[0].url")
-        div(v-else, class="no-image") No image found
-        div(class="name") {{ item.name }}
+        v-container(grid-list-md)
```

Comment:
I'm not sure exactly what `grid-list-md` does. Are we sure we want that?

Reply:
Seems that it adds a gutter between items. We might be better served by adding padding (`.pa-2`) to each `v-flex`, but maybe it does the same thing in the end. Thoughts?